lib/shared/log/to-console.js

```diff
@@ -48,10 +58,12 @@ function toConsole(log, opts) {
     })
     .forEach(function(level) {
       if (level === 'error') {
-        log.on(level, fancyLog.error);
+        log.on(level, consoleError);
       } else {
-        log.on(level, fancyLog);
+        log.on(level, consoleLog);
       }
+
+      log[level] = themingLog(theme, log[level]);
```

Comment:
I'm not sure why you are doing this but I don't think I want to override the logging functions here.

Reply:
You're right! This is wrong. This code replaces global gulplog methods, affecting other plugins that use gulplog. Instead of gulplog, we should create our own logger specific to gulp-cli. To use own logger would also eliminate the need to consider the issue #249 (and #254). 
One idea I have is to pass the namespace `Symbol('gulp-cli')` to glogg to create a new logger.

--
This is a digression and a future discussion, but could we potentially resolve #72 by creating loggers by glogg with namespace by task names and plugin names and establishing rules for plugins to use it? It’s still rough idea, and I don’t have thought of specific methods yet.

Reply:
I've mistaken. Initialization of the logging functions of gulplog v1 is  necessary, even so.